src/main/java/core/basesyntax/strategy/StorageReportGenerateImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class StorageReportGenerateImpl implements StorageReportGenerate {
+    public String getReport() {
+        StringBuilder report = new StringBuilder();
+        report.append("fruit,quantity").append(System.lineSeparator());
+        for (Map.Entry<String, Integer> entry : Storage.getAllFruits().entrySet()) {
+            report.append(entry.getKey()).append(",").append(entry.getValue())
+                    .append(System.lineSeparator());
+        }
+        return report.toString();
+
+    }
```

Comment:
The class `StorageReportGenerateImpl` should be placed in a package that reflects its purpose. Consider placing it in a package like `service.impl` as it is an implementation of a service.